src/mono/wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj

```diff
@@ -48,6 +48,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Playwright" Version="1.47.0" />
     <PackageReference Include="MSBuild.StructuredLogger" Version="2.2.350" />
+    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonToolsetVersion)" />
```

Comment:
Did `WasmBuildTests` fail without it?

Reply:
It looks like all these failures started appearing about an hour ago, which coincides with the activation of the GitHub advisory:

> **`src\mono\wasm\Wasm.Build.Tests\Wasm.Build.Tests.csproj(0,0): error NU1903: (NETCORE_ENGINEERING_TELEMETRY=Restore) Warning As Error: Package 'System.Text.Json' 6.0.0 has a known high severity vulnerability, [GHSA-8g4q-xg66-9fp4](https://github.com/advisories/GHSA-8g4q-xg66-9fp4)`**

The advisory marks this package as vulnerable, causing it to be flagged as an error during the build.